src/components/breadcrumb/_breadcrumb.scss

```diff
@@ -29,6 +29,10 @@
     color: $text-03;
   }
 
+  .bx--breadcrumb__last__item::after{
```

Comment:
@hellobrian Thank you for giving it more thought - Seems that I would have to explain what my concern is all about. With applying the new class to the breadcrumb item, consumers would have to add extra condition check, like:

```javascript
document.querySelector('bx—breadcrumb').innerHTML = myTextArray.map((text, i, a) => {
  if (i === a.length - 1) {
    return `<div class="bx--breadcrumb-item bx--breadcrumb__last__item">
      <a href="#" class="bx--link">Breadcrumb 1</a>
    </div>`;
  } else {
    return `<div class="bx--breadcrumb-item">
      <a href="#" class="bx--link">Breadcrumb 1</a>
    </div>`;
  }
});
```